packages/hoppscotch-common/src/components/app/Header.vue

```diff
@@ -281,8 +282,22 @@ const showTeamsModal = ref(false)
 const breakpoints = useBreakpoints(breakpointsTailwind)
 const mdAndLarger = breakpoints.greater("md")
 
+const bannerService = useService(BannerService)
+const banner = bannerService.content
```

Comment:
if you like destructuring, save some lines.

```suggestion
const { content: banner } = useService(BannerService)
```